### DISCUSSION TITLE
请问如何把每道题的打分结果展示出来？

### DISCUSSION THREAD

**Discussion by 123456xxn** [open-compass/opencompass#902] [Q&A]
比如在预测结果predictions里每个json文件中，给每个测试数据加个score字段？谢谢

**Comment by Leymore** [comment]
You may try the `--dump-eval-details` option. For example:

```
python3 run.py /path/to/your/config -r latest  --dump-eval-details
```

note: only some of the datasets will output correct or not info per question, if the dataset in need does not support this feature, please consider implement by yourself. You may refer to [this implement](https://github.com/open-compass/opencompass/blob/main/opencompass/datasets/math.py#L179-L196)
